initbot/bot/commands/soundboard.py

```diff
@@ -0,0 +1,125 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, Dict
+import json
+from discord.embeds import Embed  # type: ignore
+from discord.ext import commands  # type: ignore
+from discord.ext.commands import Bot  # type: ignore
+from discord.ext.commands.context import Context  # type: ignore
+from discord.player import FFmpegPCMAudio  # type: ignore
+
+
+@dataclass
+class Sound:
+    name: str
+    description: str
+    file: str
+    category: str
+
+
+sounds_file = Path(__file__).parents[2] / "sounds/sounds.json"
+if sounds_file.exists():
+    with open(sounds_file, encoding="utf8") as fd:
+        SOUNDS: List[Sound] = [
+            Sound(**a) for a in json.load(fd)["sounds"]
+        ]  # type: ignore
+
+SOUNDS_DICT: Dict[str, Sound] = {s.name: s for s in SOUNDS}
```

Comment:
Might this fail on `SOUNDS` being undefined if `sounds_file.exists()` is `false`?